tests/unittest/auto_deploy/singlegpu/transformations/library/test_kvcache_vswa_metadata.py

```
@@ -209,5 +209,22 @@ def find_swc_dep(node, visited=None):
 
+@pytest.mark.parametrize(
+    "backend, expected_cyclic",
+    [("triton_paged", False), ("trtllm", True)],
+)
+def test_vswa_sets_kernel_handles_cyclic_swa(backend, expected_cyclic):
+    """The transform records the backend's cyclic-SWA capability on the interface.
+
+    trtllm's kernel masks the sliding window internally (cyclic), so the
+    executor must pass full block tables + global lengths; triton must not.
+    """
+    gm, info, cm = _run_transform(backend=backend)
```

Ruff (0.15.15)

[warning] Unpacked variable `gm` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)